docs/weather-skills.md

```diff
@@ -79,6 +79,34 @@ aea delete my_weather_station
 aea delete my_weather_client
 ```
 
+## Communication
+This diagram shows the communication between the various entities as data is successfully sold by the car park agent to the client. 
+
+<div class="mermaid">
+    sequenceDiagram
+        participant Search
+        participant Client_AEA
+        participant Weather_AEA
+    
+        activate Client_AEA
+        activate Search
+        activate Weather_AEA
+        
+        Weather_AEA->>Search: register_service
+        Client_AEA->>Search: search
+        Search->>Client_AEA: list_of_agents
+        Client_AEA->>Weather_AEA: call_for_proposal
+        Weather_AEA->>Client_AEA: propose
+        Client_AEA->>Weather_AEA: accept
+        Weather_AEA->>Client_AEA: match_accept
+        Client_AEA->>Weather_AEA: Inform funds transfered 
```

Comment:
Transfer funds `off chain` . The key being that the transfer is not covered in the example